docs/advanced_topics/documents/title_generation_on_upload.md

```diff
@@ -1,26 +1,24 @@
-(docs_title_generation_on_upload)=
+(title_generation_on_upload)=
```

Comment:
As per other comments, we need to keep these two oages, yes they a similar but what I mean is that the updates to the script usage snippets will also be similar.

Reply:
I'll be reverting this change as the issue did not want a merge